### PR TITLE
Add Mobile menu

### PR DESCRIPTION
**Pull request by zeff96**
Added a mobile menu window to the project.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const navEl = document.querySelector('.nav-items')
+const closeEl = document.querySelector('.close-btn');
+const hamburgerEl = document.querySelector('.menu-icon');
+const linksEl = document.querySelectorAll('.nav-links');
```
Comment (Mihndim2020) [inline index.js:4]: - [ ] As [required ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md)for this project, your project is still missing the following requirements: 

 - When the user clicks (or taps) the hamburger button, the mobile menu appears.
 - When the user clicks (or taps) the close (X) button, the mobile menu disappears.
 - When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
 - When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

Please kindly implement the above changes

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @zeff96, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark:  Clean and Organized code

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

Please consider:
- [ ] updating the `Install and Usage` of your `readme file` to reflect this milestone. Please kindly update those sections with the right instruction to clone and run your project locally as this will be of help to non technical users and some of them are potential recruiters.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (zeff96) [reply index.js:4]: @Mihndim2020 
kindly explain further, from my end all those functions work as required.

Comment (zeff96) [reply index.js:4]: Hi @Mihndim2020,
kindly clarify further, I don't understand the issue apart from the readme file update.

**Review by osamaashraf6 — APPROVED** [review]
Hi @zeff96 👋, 
# STATUS: APPROVED! 🥇🟢🟢💯 


Your project is complete!  There is nothing else to say other than... it's time to merge it : ship it: 
Congratulations! 🎉 

## Highlights💯🎈🪄✔️

- [x] Good commit messages.✔️
- [x] Descriptive READme file in a professional way.✔️
- [x] Good usage in correcting GitHub flow.✔️
- [x] Nice design.✔️

## Optional suggestions

- [ ] Kindly 😊at the next milestones try to add the linter for the js at the project and follow these steps to know how to setup it [Click here](https://github.com/microverseinc/linters-config/blob/master/javascript) to read the steps💯

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**